Update index.md with restructuring of the assessment page (#12696)
mdn/content: files/en-us/learn/css/css_layout/fundamental_layout_comprehension/index.md

<table>
  <tbody>
    <tr>
      <th scope="row">Prerequisites:</th>
      <td>
        Before attempting this assessment you should have already worked through
        all the articles in this module.
      </td>
    </tr>
    <tr>
      <th scope="row">Objective:</th>
      <td>
        To test comprehension of CSS layout methods using Positioning, Flexbox, Grid and Floating.
      </td>
    </tr>
  </tbody>
</table>

## Starting point

You can download the HTML, CSS, and a set of six images [here](https://github.com/mdn/learning-area/tree/master/css/css-layout/fundamental-layout-comprehension).

Save the HTML document and stylesheet into a directory on your computer, and add the images into a folder named `images`. Opening the `index.html` file in a browser should give you a page with basic styling but no layout, which should look something like the image below.

![](layout-task-start.png)

This starting point has all of the content of your layout as displayed by the browser in normal flow.

> Alternatively, you could use a site like [Glitch](https://glitch.com/) to do your assessment. You could paste the HTML and fill in the CSS into an online editor like mentioned. Also, you will need to upload the images using the Assets functionality of your choosen online editor and replace the values in the src attributes to point to the new image locations. If the online editor you are using doesn't have a separate CSS panel, feel free to put it in a `<style>` element in the head of the document.

Reviewer: This was a little clunky. I've removed the bit about "Assets" because the name used may depend on the editor.
It also feels like perhaps this should go down in the assessment and be linked from here. However I'll leave that to you to think about rather than just merging and accepting this.
```suggestion
Alternatively, you could use a site like [Glitch](https://glitch.com/) to do your assessment, pasting in the the HTML and CSS. Note that if you use an online editor you will need to upload the images and replace the values in the `src` attributes to point to the new image locations. If the online editor you are using doesn't have a separate CSS panel, feel free to put it in a `<style>` element in the head of the document.
```

Author: Thanks for improving it - I think it's much better that way too! 🎉
Following the pattern of the other assessment pages, such a paragraph is always located at this spot (i.e. under the "Starting point" section).

Because of consistency I would leave it at this point instead of moving it further down and linking to it - linking makes it in my eyes even more complicated and unnecessary?!

But of course I will do it the way you suggested if you want to. 👍
